src/style.css

```diff
@@ -1,3 +1,70 @@
-body {
-  margin: 0;
+:root {
+  --main-text-color: #000;
+  --edit-text-color: #00ACDC;
 }
+
+  html, body {
+    width: 100%;
+    margin: 0;
+    padding:0;
+    font-family: Roboto, sans-serif;
+  }
+
+  .header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 50px;
+    font-size: 12px;
+  }
+
+  .logo {
+    height: 40px;
+    width: 40px;
+  }
+
+  .nav__list{
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    text-align: center;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    text-transform: uppercase;
+  }
+
+  .nav__link {
+    display: flex;
+    position: relative;
+    height: 60px;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    text-decoration: none;
+    color: var(--main-text-color);
+  }
```

Comment:
```suggestion
  .nav__link {
    position: relative;
    height: 60px;
    text-decoration: none;
    color: var(--main-text-color);
  }
```

Reply:
Hi, unfortunately your suggestion regarding removing flex in nav__link doesn't work.
![Screenshot 2024-03-11 at 13 33 38](https://github.com/mate-academy/layout_moyo-header/assets/108335373/f2a03d3e-b2d0-4cc8-bfa6-742200e83538)
<img width="1440" alt="Screenshot 2024-03-11 at 13 33 10" src="https://github.com/mate-academy/layout_moyo-header/assets/108335373/48c58d89-15da-4b01-a7ea-6c92c9e988d3">

Reply:
You can try just to add `text-align: center`

Reply:
<img width="1041" alt="Screenshot 2024-03-11 at 17 10 00" src="https://github.com/mate-academy/layout_moyo-header/assets/108335373/a92b8058-db96-44b3-812a-c16490b32add">
text-align: center doesn't work even if I change display to block. Regarding requirements to this task text should be centred, and height set to nav__links, flex in this case make it easy to do. And it is works right.